repos/fdbt-site/src/pages/api/multipleProducts.ts

```diff
@@ -303,7 +307,8 @@ export default (req: NextApiRequestWithSession, res: NextApiResponse): void => {
         if (numberOfProducts !== 0) {
             updateSessionAttribute(req, NUMBER_OF_PRODUCTS_ATTRIBUTE, numberOfProducts);
         }
-        const fullValidationResult = checkAllValidation(multipleProducts, isCarnet, isFlatFare);
+        const isSchool = isSchoolFareType(fareTypeAttribute);
```

Comment:
Why are we changing this? I thought this PR was just a UI test

Reply:
this was not working for a school ticket with multiple services as the "Period duration" -> "Term" is not accepting and when created the product, i used the Period duration as hour or week.